### ISSUE TITLE
Deep supervision training loss

### ISSUE THREAD

**Issue by ding3820** [282857341/nnFormer#45]
Hi, 
First of all, this is a great work with complete open source. I really appreciate it. However, the training loss with deep supervision in your released code is not aligned with what you presented in the paper. This is a bit confusing to me. 
![image](https://user-images.githubusercontent.com/28510501/148009262-ca40a67d-bc2f-4872-8df3-7deecce1b428.png)
In the paper, you apply loss on resolution of (H, W, D), (H/4, W/4, D/2) and (H/8, W/8, D/4) but the return variables below are (H, W, D) and (H/2, W/2, D/2).  
https://github.com/282857341/nnFormer/blob/e20bd3b5cd6baf2bc2c4c2b303aa21fce420e084/nnformer/network_architecture/Swin_Unet_l_gelunorm.py#L937
The implementation looks good to me but just can't connect with the paper.
